src/app/pages/Designathons/Designathon24/Designathon24.module.scss

```diff
@@ -1,160 +1,169 @@
+:root {
+	--pink: #ff75a4;
+	--hot-pink: #f84e72;
+	--peach: #ff6969;
+	--cream: #fff5e0;
+	--black: #242424;
+}
```

Comment:
Thought [non-blocking, can revisit]: for stronger compile-time guarantees, it may be better to use [SASS variables](https://sass-lang.com/documentation/variables/), but I see you do rely on the custom property values within the JSX.

Reply:
Would rather circle back to this; the JSX shouldn't be a major blocker but I'd rather tackle it in a dedicated PR. LMK if you'd pref. it to be done here.

Reply:
Agreed, can revisit later.